docs/install/windows_setup.md

```diff
@@ -141,10 +141,10 @@ To build and install MXNet yourself using [VS2017](https://www.visualstudio.com/
 1. Set the environment variable `OpenCV_DIR` to point to the OpenCV build directory that you just unzipped. Start ```cmd``` and type `set OpenCV_DIR=C:\utils\opencv\build`.
 1. If you don’t have the Intel Math Kernel Library (MKL) installed, you can install it and follow the [MKLDNN_README](https://github.com/apache/incubator-mxnet/blob/master/MKLDNN_README.md) from here, or you can use OpenBLAS. These instructions will assume you're using OpenBLAS.
 1. Download the [OpenBlas](https://sourceforge.net/projects/openblas/files/v0.2.19/OpenBLAS-v0.2.19-Win64-int32.zip/download) package. Later versions of OpenBLAS are available, but you would need to build from source. v0.2.19 is the most recent version that ships with binaries. Contributions of more recent binaries would be appreciated.
-1. Unzip the file. You can place the unzipped files and folders in another directory if you wish.
+1. Unzip the file, rename it to ```OpenBLAS``` and put it under `C:\utils`. You can place the unzipped files and folders in another directory if you wish.
 1. Set the environment variable `OpenBLAS_HOME` to point to the OpenBLAS directory that contains the `include` and `lib` directories and type `set OpenBLAS_HOME=C:\utils\OpenBLAS` on the command prompt(```cmd```).
```

Comment:
Since the file can be unzipped and placed anywhere, what is significance of C:\utils

Reply:
Yeah, it could be anywhere but original tutorial use C"\utils so I follow it. It makes users less confused?

Reply:
Usually environment variables are all caps. why is it OpenBLAS_HOME?

Reply:
not sure why OpenBLAS_HOME and OpenCV_DIR environment variables are not all cps